cylc/flow/remote.py

```diff
@@ -371,6 +371,7 @@ def _remote_cylc_cmd(
         remote command.
 
     """
+
```

Comment:
Not important but BTW there isn't supposed to be a blank line *after* the docstring for functions/methods, only classes (black will strip these out).